app/views/users/index.haml

```diff
@@ -13,10 +13,12 @@
   %tr
     %th Name
     %th Password
+    %th Customer
     %th Sites
 
   - @users.each do |user|
     %tr
       %td= link_to user.name, user
       %td= user.password
-      %td= number_with_delimiter user.sites.count
```

Comment:
We have a counter_cache column for sites.count on users? Refactored this to use it instead of calculating the count for every user record.